In [1]:
import pandas as pd
import os

# Carregamento dos arquivos CSV com as URLs

In [2]:
trt4_df_url = pd.DataFrame(columns=[ 'link' ])

pasta = "urls"

for nome_arquivo in os.listdir( pasta ):
    print( nome_arquivo )
    df = pd.read_csv( pasta+"\\"+nome_arquivo, encoding='utf-8' )
    df=df.drop('_type',1)
    trt4_df_url = trt4_df_url.append( df )

trt4_df_url.info()

TRT4_urls_2017-01-01_2017-05-01.csv
TRT4_urls_2017-05-01_2017-12-14.csv
TRT4_urls_2017-12-14_2018-10-08.csv
TRT4_urls_2018-10-08_2019-05-08.csv
TRT4_urls_2019-05-08_2019-11-19.csv
TRT4_urls_2019-11-19_2019-12-31.csv
<class 'pandas.core.frame.DataFrame'>
Int64Index: 140554 entries, 0 to 6568
Data columns (total 1 columns):
link    140545 non-null object
dtypes: object(1)
memory usage: 2.1+ MB


### Cálculo do tamanho da amostra
<br> Para estar certo da representatividade de decisões judiciais publicadas em 99% das vezes e que a amostra estatística extraída esteja no máximo 1% abaixo ou 1% acima da média da população, a quantidade deve ser de no mínimo 14.880. Optou-se por extrair quantidade maior para haver observações sobressalente no caso de algumas serem necessárias ser excluídas em fases posteriores de processamento.

### Tamanho da população: 140554
### Tamanho da amostra: 14.880 + 20% = 17.856

In [3]:
trt4_df_url.sample( 30000, random_state=1 ).to_csv( "TRT4_df_url_2017_2018_2019_amostra_rndstate_1.csv", encoding='utf-8', index=False )